lib/browser/cookie.js

```diff
@@ -66,19 +63,24 @@ exports.setCookies = util.deprecate(function setCookies(cookies) {
 
 exports.setCookieValues = function setCookieValues(cookies) {
   const browser = this;
+
   return Promise.all(
-    _.map(cookies, (value, name) => {
-      return browser.setCookieValue(name, value);
+    Object.entries(cookies).map(pair => {
+      return browser.setCookieValue(...pair);
```

Comment:
minor nit: while this is clever, if you ever end up wanting to type check this code, it'll work better to have something more explicit like:
```suggestion
    Object.entries(cookies).map(([k, v]) => {
      return browser.setCookieValue(k, v);
```